### ISSUE TITLE
Publish build for sbt 1.0.0

### ISSUE THREAD

**Issue by unoexperto** [sbt/sbt-native-packager#815]
Guys, could you please build and publish build for sbt 1.0.0-M4 or newer version?

**Comment by muuki88** [comment]
What are the issues you are facing when running native-packager with sbt 1.0.0-M4?

**Comment by cppexpert** [comment]
It fails to get the plugin.

```
[warn] 	Note: Unresolved dependencies path:
[warn] 		com.typesafe.sbt:sbt-native-packager:1.1.1 (scalaVersion=2.11, sbtVersion=1.0.0-M4) (/home/expert/work/sideprojects/console/project/plugins.sbt#L3-4)
```

**Comment by muuki88** [comment]
Ah, okay. Currently we don't have any infrastructure to publish for multiple SBT versions.
If you would like to make a pull request for this, we can publish native-packager for 1.0.0-M4.

**Comment by norcalli** [comment]
SBT 1.0.0 has been officially released, so this seems like a higher priority now.

**Comment by muuki88** [comment]
Already done. Thanks for notifying, I must have overlooked this ticket. `v1.2.2` is released against SBT 1.0.
